ChangeLog.md

```diff
@@ -18,8 +18,8 @@ to browse the changes between the tags.
 
 See docs/process.md for more on how version tagging works.
 
-Current Trunk
--------------
+2.0.20
+------
```

Comment:
It would be good to mention that from here the value in `emscripten-version.txt` in git is forward-looking etc.

I feel like we should document this elsewhere too - perhaps on the "developer guide" page on the website?

Reply:
I added a description to the ChangeLog.  I honestly can't think of any use case that might be effected by this?  At least none of our processes were.   I'm not sure who it would benefit to document this outside of where we already have.

Reply:
The risk is low, I agree. But one possible issue is people bisecting on the emscripten repo and looking at the version file as they go. I think it's useful to mention that file's meaning has changed at this time.

Reply:
I have mentioned it in this ChangeLog entry..  do you think I should mention it in the bisect instructions too?

Reply:
Maybe not needed in bisection. The docs there focus on multirepo bisection, which this should not affect.

If there is confusion we can update the docs accordingly later.